lib/tasks/stop_data_convert_and_import.rake

```diff
@@ -29,4 +29,15 @@ namespace :bus_stops do
       stop.update_attributes(new_attributes)
     end
   end
+  task import_shelter_type_data: :environment do
+    csv = CSV.read('all_stop_data.csv', headers: true)
+    csv.each do |row|
+      shelter_type = {}
+      shelter_type['shelter_type'] = case row['shelter_type']
```

Comment:
This case statement evaluates to `nil`whenever `row['shelter_type']` is _not_ `'Modern'`. I'm pretty sure this will end up erasing the shelter type on all non "Modern" shelters.

Reply:
Oh, yeah that does happen when I go through step by step. When I ran the rake task on sample data, I didn't see all non Modern shelter types turn to nil. 
Prior to the rake task:
```
 2.5.0 :030 > BusStop.all.pluck(:shelter_type)
   (0.5ms)  SELECT `bus_stops`.`shelter_type` FROM `bus_stops`
 => ["Modern full", "Wooden", "Modern", "No shelter", "Modern full", "Modern half", 
"Modern full", "Modern", nil, nil, nil, "Extra large", "Wooden", nil]
```
After running the rake task:
```
2.5.0 :001 > BusStop.all.pluck(:shelter_type)
   (0.4ms)  SET  @@SESSION.sql_mode = CONCAT(CONCAT(@@sql_mode, ',STRICT_ALL_TABLES'), ',NO_AUTO_VALUE_ON_ZERO'),  @@SESSION.sql_auto_is_null = 0, @@SESSION.wait_timeout = 2147483
   (0.4ms)  SELECT `bus_stops`.`shelter_type` FROM `bus_stops`
 => ["Modern full", nil, "Modern full", "No shelter", nil, "Modern half",
nil, "Modern full", nil, nil, nil, "Extra large", "Wooden", nil]
```
In this case it erased some non Modern shelter types and went unnoticed. Either way, that's not what we want. I'll make sure to fix this. Thanks